crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -296,6 +296,24 @@ def transition(inst, name, state)
     [200, { name: name }]
   end
 
+  # simple check if HA clusters report some problems
+  def check_cluster_health
+    cluster_founders = NodeObject.find("pacemaker_founder:true AND pacemaker_config_environment:*")
+    return if cluster_founders.empty?
+    check_if_nodes_are_available cluster_founders
+    nodes_with_failed_actions = []
+    cluster_founders.each do |n|
+      ssh_retval = n.ssh_retval('crm status | grep "^Failed actions:"')
+      nodes_with_failed_actions.push n.name if ssh_retval
+    end
+    unless nodes_with_failed_actions.empty?
+      @logger.error("failed actions on clusters nodes: #{nodes_with_failed_actions.join(', ')}
+Clean pacemaker resources using 'crm resource cleanup' before proceeding with the upgrade.")
```

Comment:
I suspect Hound is silently complaining about this, and there's no good reason to break the indentation here. Please can you fix it? Thanks!